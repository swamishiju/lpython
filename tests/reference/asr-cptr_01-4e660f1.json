{
    "basename": "asr-cptr_01-4e660f1",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cptr_01.py",
    "infile_hash": "92647f6c175f3599be307df700902b5f7fe69826f22fd5f082713ee4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-cptr_01-4e660f1.stderr",
    "stderr_hash": "b75484482c47172cb76a64d8091833a284305788241ef59c8d6efb06",
    "returncode": 2
}