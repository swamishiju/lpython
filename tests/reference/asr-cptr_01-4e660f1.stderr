semantic error: Function 'empty_c_void_p' is not declared and not intrinsic
 --> tests/errors/cptr_01.py:4:15
  |
4 |     x: CPtr = empty_c_void_p()
  |               ^^^^^^^^^^^^^^^^ 
